src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -331,7 +339,11 @@ impl DdlService for DdlServiceImpl {
 
         let version = self
             .ddl_controller
-            .run_command(DdlCommand::CreateStreamingJob(stream_job, fragment_graph))
+            .run_command(DdlCommand::CreateStreamingJob(
+                stream_job,
+                fragment_graph,
+                false,
```

Comment:
Creating an index is also a DDL job. I think we should pass `run_in_background` to this method as well.